requirements.txt

```diff
@@ -22,3 +22,4 @@ six==1.15.0
 shodan==1.25.0
 pycurl==7.43.0.5
 xmltodict==0.12.0
+icmplib
```

Comment:
Do mention the current version(or tested working version) of this package. So, dependbot push update of that package.